addons/mail/static/src/discuss/call/public_web/discuss_sidebar_call_participants.js

```diff
@@ -60,11 +60,11 @@ export class DiscussSidebarCallParticipants extends Component {
     get sessions() {
         const sessions = [...this.props.thread.rtcSessions];
         return sessions.sort((s1, s2) => {
-            const persona1 = s1.channelMember.persona;
-            const persona2 = s2.channelMember.persona;
+            const persona1 = s1.channelMember?.persona;
+            const persona2 = s2.channelMember?.persona;
             return (
                 persona1?.name?.localeCompare(persona2?.name) ||
-                s1.channelMember.id - s2.channelMember.id
+                s1.channelMember?.id - s2.channelMember?.id
```

Comment:
Wait, you can't subtract `undefined`:

![image](https://github.com/user-attachments/assets/53b38e87-75d4-4138-afd9-f0b131dfc69d)